apps/agreement/contracts/Agreement.sol

```diff
@@ -315,18 +315,21 @@ contract Agreement is IAgreement, AragonApp {
     /**
     * @notice Close action #`_actionId`
     * @dev If allowed by the originating Disputable app, this function allows users to close actions that are not:
-    *      - Closed
     *      - Currently challenged
     *      - Ruled as voided
     *      - Ruled in favour of the submitter
+    *      It ignores if the action is already closed, without re-closing it
     *      Initialization check is implicitly provided by `_canClose()` as disputable actions can be created only
     *      via `newAction()` which already requires initialization implicitly through `activate()`
     * @param _actionId Identification number of the action to be closed
     */
     function closeAction(uint256 _actionId) external {
         Action storage action = _getAction(_actionId);
-        require(_canClose(action), ERROR_CANNOT_CLOSE_ACTION);
+        if (action.closed) {
+            return;
```

Comment:
To double check, we were discussing whether we should return a flag here to mark if you had actually closed the action in this request or not.

However, this is actually not that useful:

1. We only unlock tokens from the staking pool, so the caller usually doesn't receive any tokens directly from this call
2. For a contract that really wanted to know, the `closed` status is available from `getAction()` and should be called first (similar to the PR in aragonOS that we closed)

Reply:
Yea, I finally decided no to add it to be honest.